scripts/setup.sh

```diff
@@ -9,14 +9,12 @@ export AUTHJS_VERSION=""
 export REGISTRY_REPO="npm-topic"
 export REGISTRY="${ARTIFACTORY_URL}/api/npm/${REGISTRY_REPO}"
 setup_service node v14.18.2
-# Use the cacert bundled with centos as okta root CA is self-signed and cause issues downloading from yarn
-setup_service yarn 1.21.1 /etc/pki/tls/certs/ca-bundle.crt
+setup_service yarn 1.22.19 
```

Comment:
can we change this to
```
setup_service node-and-yarn "14.18.2" "1.22.19"
```

Reply:
This will put us on our newest way to install `yarn` as well

Reply:
Build fails with this error when I use `node-and-yarn` 
```
2023-03-22 21:26:38 Expected to find .nvmrc at /root/okta/okta-signin-widget/.nvmrc
2023-03-22 21:26:38 Repo must have its own .nvmrc file for this entry point.
```
We can tackle this as a separate issue I guess. Will go ahead and merge without this change.

Reply:
Well... Turns out, I'll just need to add an `.nvmrc` file in the root directory with the version. Will try that and see if it works.